### PR TITLE
added methods for mutating vertical tabs

### PR DESCRIPTION
**Pull request by ifellows**
Adds features discussed in #148 .

### CODE REVIEW

File: inst/www/vertical-tab-panel/vertical-tab-panel.js [hunk 1]
```diff
@@ -4,7 +4,7 @@
 /* Source: https://bootsnipp.com/snippets/featured/simple-vertical-tab */
 
 $(document).ready(function() {
-    $("div.bhoechie-tab-menu>div.list-group>a").click(function(e) {
```
Comment (ifellows) [inline inst/www/vertical-tab-panel/vertical-tab-panel.js]: Note: I believe that without this edit, verticalTabsetPanel will not work if inserted via insertUI.

**Comment by ifellows** [comment]
@pvictor : Any thoughts on this?

**Comment by pvictor** [comment]
Thanks @ifellows ! Great work! Sorry for the long delay.
I added you to the DESCRIPTION & updated NEWS.

**Comment by pvictor** [comment]
On CRAN now !

**Pull request merged**